lib/Twig/Error.php

```diff
@@ -188,10 +188,10 @@ protected function guessTemplateInfo()
         $template = null;
         $templateClass = null;
 
-        if (version_compare(phpversion(), '5.3.6', '>=')) {
-            $backtrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS | DEBUG_BACKTRACE_PROVIDE_OBJECT);
-        } else {
+        if (PHP_VERSION_ID < 50306) {
             $backtrace = debug_backtrace();
+        } else {
+        	$backtrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS | DEBUG_BACKTRACE_PROVIDE_OBJECT);
```

Comment:
Please don't change the order of the code to avoid unnecessary diffs (and don't use tabs for the indentation)